src/main/java/uk/gov/digital/ho/hocs/document/dto/CreateDocumentRequest.java

```diff
@@ -6,7 +6,6 @@
 
 import java.util.UUID;
 
-@AllArgsConstructor
```

Comment:
Could have got rid of the existing constructor and kept this - I'm not overly fussed though.